types/chance/index.d.ts

```diff
@@ -225,13 +236,16 @@ declare namespace Chance {
         name: string;
         abbreviation: string;
     }
-}
 
-// window.chance
-declare var chance: Chance.Chance;
-declare var Chance: Chance.ChanceStatic;
```

Comment:
these two lines assume that chance is installed globally for everyone. Deleting it as typescript thing is fine to use `chance` even if it is not global